Fixed suggestions about getting ES8336 quirk default. (#477)
thesofproject/sof-docs: getting_started/intel_debug/suggestions.rst

   #define SOC_ES8336_HEADSET_MIC1		BIT(8)

> The default and actual quirk values for the platform can be got

Author: fixed

> from the dmesg log file (/var/log/dmesg) in the line like:

Reviewer: There is `/var/log/dmesg` file on Fedora or Arch Linux, it's distro-specific.

Don't recommend the `dmesg` command either because it's a ring buffer that can run out.

There are countless pages on the Internet that explain what each distro does with kernel logs, just let users find that themselves:
```suggestion
in the kernel logs in a line that looks like:
```

Author: fixed

> from the dmesg log file (/var/log/dmesg) in the line like:

Author: fixed

.. code-block:: txt

    ...
    sof-essx8336 sof-essx8336: quirk mask 0x1a0
    ...

for the unchanged mask, or:

> .. code-block:: txt

Reviewer: This is invalid for some reason:

https://github.com/thesofproject/sof-docs/actions/runs/6297068973/job/17095424648?pr=477

```
/home/runner/work/sof-docs/sof-docs/getting_started/intel_debug/suggestions.rst:300: WARNING: Pygments lexer name 'txt' is not known
/home/runner/work/sof-docs/sof-docs/getting_started/intel_debug/suggestions.rst:308: WARNING: Pygments lexer name 'txt' is not known

```

Author: the code-block fixed to `none`.

    ...
    sof-essx8336 sof-essx8336: Overriding quirk 0x1a0 => 0x120
    ...

for the quirk overriding.

> The overriden quirk value can also be got from the

Author: fixed

/sys/module/snd_soc_sof_es8336/parameters/quirk (the value is reported

> as plain integer, not hexadecimal). If the quirk is not overriden,

Author: fixed

the `-1` value is returned.

Changes to the default can be added with the following option in
e.g. /etc/modprobe.d/alsa-base.conf. Only the bits listed above can be

> modified, others need to be kept as is.

Author: fixed
